gmso/core/topology.py

```diff
@@ -746,6 +747,171 @@ def get_index(self, member):
 
         return index
 
+    def to_datatables(self, parameter="sites", labels=None, unyts_bool=True):
```

Comment:
agree on this. Also i think the `labels` can be a bit confusing since it's specific to attributes of sites and not other object, so may be change this to be `site_attribs`?

Reply:
Agreed, it is now changed to site_attrs.